sentences/fr/homeassistant_HassTurnOff.yaml

```diff
@@ -7,6 +7,11 @@ intents:
           - <eteins> [<le>]{name} <dans> [<le>]{area}
         excludes_context:
           domain:
+            - binary_sensor
             - cover
+            - lock
+            - scene
+            - script
+            - sensor
             - vacuum
-            - valve
+            - valve
```

Comment:
Add back the empty line to make the linter happy :) 
Your tests are not passing for cosmetic issues

Reply:
Thank you, I struggled a little to fix it :)